joda-time/src/main/java/org/joda/time/field/PreciseDurationField.java

```diff
@@ -62,7 +62,7 @@ public final boolean isPrecise() {
      */
     @Override
     public final long getUnitMillis() {
-        return iUnitMillis;
+        return this.iUnitMillis;
```

Comment:
This changes code needs more review

Reply:
@Asib27 please change only the formatting and not other code aspects, even when keeping the code equivalent. Why would we need `this.` here?